walkthrough/05-aws/04-reusing-stacks/live/dev/terragrunt.stack.hcl

```diff
@@ -0,0 +1,5 @@
+stack "stateful_service" {
+	source = "../../stacks/stateful-service"
+	path   = "stateful-service"
+}
```

Comment:
It's confusing that we use the file name `terragrunt.stack.hcl` for both an _instance_ and the _definition_ of the stack. I wonder if we should introduce a convention like the following:

- `terragrunt.stack.hcl` and `terragrunt.stack.def.hcl` are both valid stack file names
- We recommend using `terragrunt.stack.def.hcl` to define your Stack definitions, though you can use them for whatever you want.
- We might do something similar with units (e.g. `terragrunt.def.hcl`)

Reply:
I don't see what that buys us.

Users are already using OpenTofu/Terraform, where `.tf` files are used both for module files and instantiating modules. I don't think we need to complicate matters, at least not for the initial release, if a more minimal implementation with one valid filename will do the job just as well.